src/admin/components/elements/ReactSelect/MultiValue/index.tsx

```diff
@@ -53,7 +53,6 @@ export const MultiValue: React.FC<MultiValueProps<Option>> = (props) => {
         onMouseDown: (e) => {
           if (!disableMouseDown) {
             // we need to prevent the dropdown from opening when clicking on the drag handle, but not when a modal is open (i.e. the 'Relationship' field component)
-            e.preventDefault();
```

Comment:
@AlessioGr we need to be sure that the comment above is affected by removing this line. It was put here intentionally.

Reply:
@jacobsfletch The stopPropagation() below seems to be enough. The drawer is not opening when you click on the handle.